.rubocop.yml

```diff
@@ -81,3 +81,11 @@ Metrics/CyclomaticComplexity:
 Metrics/PerceivedComplexity:
   Max: 13
 
+Rails/Exit:
+  Exclude:
+    - 'lib/tasks/task_helpers.rb'
```

Comment:
The file shouldn't be a rake task (otherwise, it will be picked by the Rakefile) but instead a regular ruby file.
Rubocop then lint it as if it was regular rails code therefore complaining about the usage of `exit` and direct usage of standard streams.